### ISSUE TITLE
Version dropdown not available when using tabs feature

### ISSUE THREAD

**Issue by ppanero** [zayd62/mkdocs-versioning#49]
When the [tabs feature](https://squidfunk.github.io/mkdocs-material/getting-started/#tabs) is used. I.e. setting the `mkdocs.yaml` file as:

``` yaml
theme:
  [...]
  feature:
    tabs: true
```

The version dropdown is not displayed since it is changed to the top menu (see screenshot):

![Screenshot 2020-03-30 at 09 29 56](https://user-images.githubusercontent.com/6756943/77887554-148be100-726b-11ea-97c5-64ec74a6362f.png)

It is a platform independent but.

**Comment by zayd62** [comment]
well. initial solution is to have a check for tabs and remove it when building but since tabs is a theme specific option, adding checks for these edge cases kind of defeats the point of the plugin being "theme agnostic" (unless tabs is a feature in the mkdocs built in themes).

A fix is quite simple but it would mean that all the version will not be neatly inside a dropdown. i personally dont mind having the version not neatly inside a drop down. maybe it could be a config option.  

This is what it currently looks like

![image](https://user-images.githubusercontent.com/26032382/77939568-4a52b900-72af-11ea-9c54-484d2b669494.png)

> notice the little arrow next to the text **Version Select**. this makes the list collapsible

 this is what a fix would look like

![image](https://user-images.githubusercontent.com/26032382/77939758-8e45be00-72af-11ea-81db-0a26ddcbfdff.png)

> notice no arrow

The fix is quite simple () however, adding a config option will take a little bit longer to implement since i will need to document it and the documentation is currently garbage and needs a serious rewrite something I dont have a lot of time for right now.

# the fix

https://github.com/zayd62/mkdocs-versioning/blob/e85bdad046e82d8f4aae0bf72dfe26ac5a5cfa8d/mkversion/version.py#L76-L84

this section of code in `version.py` is changed to

```python

    # building paths for each version
    # version_path_pair = []
    for i in built_docs:
        nav_item = {}
        nav_item[i] = i + '/'
        nav.append(nav_item)
    # versiondict = {'Version Select': version_path_pair}
    # nav.append(versiondict)
```

let me know if the fix is working for you. Ill most likely upload the fix to pypi once I have fixed your #48

**Comment by zayd62** [comment]
hey @ppanero, fixed this issue, updated documentation and made it available on pypi. latest version is 0.3.0. recommend reading through the documentation https://zayd62.github.io/mkdocs-versioning/ as there are significant changes
